app/services/idv/steps/ipp/verify_step.rb

```diff
@@ -19,6 +17,14 @@ def extra_view_variables
             step_url: method(:idv_in_person_step_url),
           }
         end
+
+        private
+
+        def add_proofing_component
+          ProofingComponent.
+            create_or_find_by(user: current_user).
+            update(document_check: DocAuth::Vendors::USPS)
+        end
```

Comment:
It occurs to me that the way we consider ProofingComponent is per-user, rather than per-profile. This also despite the fact that a profile includes a `proofing_components` JSON field:

https://github.com/18F/identity-idp/blob/7da2cf035ba2f5b223b37cc08d7ee50a52188964/db/schema.rb#L432

There's a lot of precedent for this, though I wonder if it's correct, since technically a user could have several profiles, each of which would have some associated components.

Reply:
the proofing component on the profile takes a copy/snapshot of the user-wide one I think? Provides some evidence of change over time

Reply:
Ah, interesting. I didn't realize it was doing that, and I can see how that could work fine. Maybe not an issue after all.

Reply:
🤷 I am not sure why it was done this way but just for reference: https://github.com/18F/identity-idp/blob/7da2cf035ba2f5b223b37cc08d7ee50a52188964/app/services/idv/profile_maker.rb#L11-L26